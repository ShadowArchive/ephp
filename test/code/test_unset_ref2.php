<?php

$b = 1;
print '$b = ';
var_dump($b);
$a = [ &$b, 2, 3 ];
print '$a = ';
var_dump($a);
print 'we remove $b' . "\n";
unset($b);
print '$a = ';
var_dump($a);
print '$b = ';
var_dump($b);

