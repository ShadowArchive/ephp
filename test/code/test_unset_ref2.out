$b = int(1)
$a = array(3) {
  [0]=>
  &int(1)
  [1]=>
  int(2)
  [2]=>
  int(3)
}
we remove $b
$a = array(3) {
  [0]=>
  int(1)
  [1]=>
  int(2)
  [2]=>
  int(3)
}
$b = 
Notice: Undefined variable: b in {{CWD}}/test/code/test_unset_ref2.php on line 14
NULL
